### ISSUE TITLE
Support different vertical hydraulic conductivity profiles

### ISSUE THREAD

**Issue by verseve** [Deltares/hydromt_wflow#254]
### Kind of request

Adding new functionality

### Enhancement Description

Different vertical hydraulic conductivity for the `SBM` concept are now supported in Wflow.jl. It would be useful to add this option also to hydromt_wflow. The different options are described in [The-SBM-soil-water-accounting-scheme](https://deltares.github.io/Wflow.jl/dev/model_docs/vertical/sbm/#The-SBM-soil-water-accounting-scheme) and [Subsurface flow routing](https://deltares.github.io/Wflow.jl/dev/model_docs/lateral/kinwave/#Subsurface-flow-routing).

### Use case

See https://github.com/Deltares/Wflow.jl/issues/313.

**Comment by hboisgon** [comment]
I think now that we have the method to resample the c parameter for different wflow layers, this should be relatively straightforward to do and apply to ksatver as well. I guess this would have an influence on the paddy / irrigation workflow in #226  as we would have to update also ksatver and not just c right?
Then maybe we need a bit of restructure to separate the parameters defined for the wflow soil depths and the others.

**Comment by verseve** [comment]
> I think now that we have the method to resample the c parameter for different wflow layers, this should be relatively straightforward to do

Yes, I agree, the implementation of this functionality is relatively straightforward. I think it would be good to also include an additional method to estimate the soil depth (now from SoilGrids with a max depth of  2 m). These ksat profiles have also been added to wflow for soil depths for which soil measurements are not available. For example this method from wflow_sbm.py: https://github.com/openstreams/wflow/blob/master/wflow/wflow_sbm.py#L1923.
